flexmeasures/data/models/charts/belief_charts.py

```diff
@@ -1155,7 +1155,14 @@ def create_chargepoint_session_chart(
             },
         ],
     }
-
+    for idx, entry in enumerate(sensors_to_show):
+        title = entry.get("title")
+        if title == "Power flow by type":
```

Comment:
Also here you hardcode a graph title, which we never programmatically use in FlexMeasures (we do so in a plugin, but FlexMeasures main code cannot depend on plugins).
We need some other way to do this, maybe a config setting for users with charging sessions?